### ISSUE TITLE
OpenTok Android SDK update

### ISSUE THREAD

**Issue by sking-j2interactive** [opentok/cordova-plugin-opentok#196]
# Feature Request

**Description**
The current version of the OpenTok Android SDK referenced in this repo is 2.15.3 which is not supported by Vonage any longer.

**Proposal**
Please upgrade to version 2.18.1 

**Links / references**
I've attached an email I got from Vonage Support regarding this.
[Vonage API Support support ticket #1530919: Cordova Android app is crashing after connecting to session.eml.zip](https://github.com/opentok/cordova-plugin-opentok/files/5404817/Vonage.API.Support.support.ticket.1530919.Cordova.Android.app.is.crashing.after.connecting.to.session.eml.zip)

**Comment by CaioMelo8** [comment]
I wonder if this is why the app I'm currently developing has suddenly started failing delivering both video and audio from the subscriber stream.

The app, which is stuck at the 2.15.3 version of the Opentok client, connects to the stream created by a client website, which in turn uses the 2.18.x version There could be some incompatibility between those versions.
What's weird is that the issue doesn't happen every single time, but it consistently occurs when accessing the client website from specific machines.

I might try to update the Opentok client version on my fork repo to see if it could solve this issue.

**Comment by CaioMelo8** [comment]
Yep, updating the SDK did the trick. In my case, I updated `@opentok/client` to 2.19.2 and native Android/iOS SDKs to 2.19.0.

 I'm sending a PR for this, it'd be really nice if it could be reviewed and merged.
